src/stories/css/input.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+    font-family: 'VisbyCF-ExtraBold';
+    src: url('../assets/Fonts/VisbyCF/VisbyCF-ExtraBold.ttf')  format('truetype');
+  }
+@font-face {
+font-family: 'OpenSans';
+src: url('../assets/Fonts/Open_Sans/OpenSans-Regular.ttf')  format('truetype');
+}
+input{
+    font-family: 'OpenSans', sans-serif;
+    font-weight: 400;
+    outline: none;
+    border: none;
+}
+
+.input-icon--left{
+    width: 20px;
+    height: auto;
+    padding-right: 9px;
+}
+.input-icon--right{
+    width: 20px;
+    height: auto;
+    padding-right: 9px;
+}
+.storybook-input-div {
+    font-family: 'OpenSans', sans-serif;
+    display: flex;
+    flex-direction: column;
+    font-weight: 400;
+    width: fit-content;
+    flex-basis: auto;
+
+}
+.storybook-input-label {
+    padding-bottom: 8px;
+}
+.storybook-input {
+    font-family: 'OpenSans', sans-serif;
+    width: auto;
```

Comment:
I'd set this to a consistent `400px` length to stay consistent with the mocks. Adjust font/icon sizes as need be to match this convention.

Reply:
Do you mean 240px? I don't see 400px on the mockups.

Reply:
My bad, I meant 240px. Sorry 🥺